services/frontend-v3/src/components/experience/ExperienceView.jsx

```diff
@@ -11,6 +11,32 @@ const ExperienceView = ({ experienceInfo }) => {
       backgroundColor: "#007471",
     },
   };
+  const getUrl = (item) => {
+    if (item.attachmentLinks && item.attachmentLinks.length > 0)
+      return item.attachmentLinks.map((i) => (
+        <Tag color="rgb(114, 114, 114)" key={i.id}>
+          <LinkOutlined />
+          <a target="_blank" rel="noreferrer" href={i.url}>
+            {i.name}
+          </a>
+        </Tag>
```

Comment:
Can you put the a tag outside and add a surround the text with a span - for it to be like the education ones. For it to have a pointer cursor, you'll probably need to add `style={{cursor: 'pointer'}}` to the tag, and same thing for the education link